Library/MeterImage.cpp

```diff
@@ -170,6 +183,12 @@ bool MeterImage::Update()
 			{
 				if (m_ImageName.empty())
 				{
+					auto plugin_measure = dynamic_cast<MeasurePlugin *>(m_Measures[0]);
+					if (plugin_measure != nullptr && plugin_measure->HasImageData())
+					{
+						LoadImageFromPluginMeasure(plugin_measure);
+						return true;
+					}

```

Comment:
I guess this block lacks some safety checks, but idk what's missing, because i still didn't experiment with this feature.

Reply:
As I commented above, `plugin_measure->HasImageData()` is not a valid check.
`LoadImageFromPluginMeasure` should return `bool` indicating whether the image was actually loaded, and if it wasn't, then we should fallback to loading file from measure string value.